docs/docs/command-line-flags.md

```diff
@@ -338,26 +338,65 @@ The Web Site flags control the behavior of Marathon's web site, including the us
 
 ### Metrics Flags
 
-* <span class="label label-default">v0.13.0</span> `--[disable_]metrics` (Optional. Default: enabled):
-    Expose the execution time per method via the metrics endpoint (/metrics) using code instrumentation.
-    Enabling this might noticeably degrade performance but it helps finding performance problems.
-    These measurements can be disabled with --disable_metrics. Other metrics are not affected.
-* <span class="label label-default">v1.6.0</span> `--metrics_averaging_window` (Optional. Default: 30 seconds):
-    Configure the size of the sliding average window that is used to compute the values for the `/metrics` endpoint. Note that this value
-    should be at least double the value of the `kamon.metric.tick-interval` parameter, that is 1 second by default.
-* <span class="label label-default">v0.13.0</span> `--reporter_graphite` (Optional. Default: disabled):
-    Report metrics to [Graphite](http://graphite.wikidot.com) (StatsD) as defined by the given URL.
+* <span class="label label-default">v1.7.0</span> `--metrics_name_prefix`:
+    Configure the prefix that is used when constructing metric names (default: marathon).
```

Comment:
does this support namespace name?  like `dcos.marathon` or `root.marathon`?

Reply:
Yes.